source/core/assets/levels/earth/main-area/entities.json

```diff
@@ -6,28 +6,27 @@
       "y": 67
     }
   },
-  "enemies": [
+  "spawners": [
     {
-      "health": 20,
-      "baseAttack": 10,
-      "speed": 5,
-      "behaviour": "PTE",
-      "spritePath": "images/enemy/base_enemy.atlas",
+      "wave1" : [
+        10,
+        0,
+        0
```

Comment:
Consider making each value map to a descriptive name to make it easier to modify and add to the config files.